### ISSUE TITLE
Using Typescript for the project

### ISSUE THREAD

**Issue by BrutalCSkakan** [fhqvst/avanza#71]
Hello! I am fully aware of that this isn't an issue for the repo itself, however I have a hard time using this project together with my already existing Typescript/Angular project. 

Is there anyone else that has issues referencing to this Library, and that might have solved this?

**Comment by fhqvst** [comment]
Yeah types would be nice. We're already able to auto-generate docs, so perhaps we can autogen types too. There's also this fork which contains some types:
https://github.com/codler/avanza-api

**Comment by shvetsjr** [comment]
Any update on this? :)
